articles/active-directory/develop/v2-oauth2-device-code.md

```diff
@@ -30,6 +30,8 @@ The entire device code flow looks similar to the next diagram. We describe each
 
 ![Device code flow](./media/v2-oauth2-device-code/v2-oauth-device-flow.svg)
 
+[!NOTE] As shown in the above diagram, the client which receives the tokens is a different machine than the user_agent where authentication is performed. If you are using facilities such as device registration, Intune policies, conditional access etc. that are applicable to the user_agent, be mindful that the tokens are being sent to a different machine (the client) that may not meet those conditions.
```

Comment:
@maliksahil Avoid the use of alert boxes, and _especially_ avoid putting alert boxes back-to-back with other prominent visual elements like images as it hides them and their content even more.

Please move this content to its own section or otherwise convert it to normal text (not an alert box). Also, please open your PR in the private repo so we can see it on staging.

From my style guide: 

## Avoid alert boxes

**Use alert boxes sparingly** (Note, Important, Information, etc.)

They can be used effectively, but **use alert boxes very sparingly and judiciously**. The fewer alert boxes we have in our doc set, the more effective they will be when we do use them.

It's OK to use them sparingly, but there are two primary reasons to avoid them when possible:

- We've seen in UX testing that readers often *skip* the content in these alert boxes, counter to their intent.
- Use of alert boxes fragments the flow and scanability of a doc, essentially making it more difficult to read.

Both of these issues are especially problematic when there's an overuse of alert boxes in content, causing "box fatigue." Our doc set previously had an extreme overuse of alert boxes, but thanks to earlier efforts in integrating them into article bodies, we are now in a much better state.